pkg/util/splitter.go

```diff
@@ -0,0 +1,65 @@
+package util
```

Comment:
this file is a modified version of https://github.com/marccampbell/yaml-toolbox/blob/master/pkg/splitter/splitter.go